doc/radosgw/multisite.rst

```diff
@@ -400,7 +400,7 @@ section of the guide. The configuration section for radosgw should resemble::
 
   [client.rgw.us-east-1]
   rgw_frontends="civetweb port=80"
-  rgw_zone=us-east
+  rgw_zone=us-east-1
```

Comment:
the zone name is us-east but the section name is wrong and should be client.rgw.us-east